logdna/index.js

```diff
@@ -14,7 +14,10 @@ const REQUEST_RETRY_INTERVAL = parseInt(process.env.LOGDNA_REQUEST_RETRY_INTERVA
 
 // Get Configuration from Environment Variables
 const getConfig = () => {
-    let config = {};
+    const pkg = require('./package.json');
+    let config = {
+        UA: `${pkg.name}/${pkg.version}`
```

Comment:
What's wrong with spelling out `UserAgent`?

Reply:
Nothing wrong - I was just following what was implemented for `logdna-agent:v1.5.6`

Reply:
but for clarity and readability, `UserAgent` is da best